packages/mobile/src/notifications.ts

```diff
@@ -62,21 +62,21 @@ class PushNotifications {
 
   onNotification = (notification: any) => {
     console.info(`Received notification ${JSON.stringify(notification)}`)
-    if (notification.userInteraction || Platform.OS === 'android') {
-      track(
-        make({
-          eventName: EventNames.NOTIFICATIONS_OPEN_PUSH_NOTIFICATION,
-          ...(notification.message
-            ? {
-                title: notification.message.title ?? notification.title,
-                body: notification.message.body ?? notification.message
-              }
-            : {})
-        })
-      )
-
-      this.navigation?.navigate(notification.data.data)
-    }
+    // NOTE: Removed the check for userInteraction
+    // It was not being set to true when the user pressed the push notification
```

Comment:
Does the app navigate when it's open and a new push notif comes in now? Seems like the behavior is different on ios and android

Reply:
Yea, I'll have to check this.

Reply:
It was, compromise seems to be checking if the app is in the foreground or background when the notif comes in

Reply:
Sounds like a plan!